packages/l10n/src/time-zone-information.ts

```diff
@@ -1,40 +1,67 @@
-import type { TimeZones } from './types';
+import type { TimeZones, TimeZoneTranslations } from './types';
 
 import * as PropTypes from 'prop-types';
+import moment from 'moment-timezone';
 import { createL10NInjector, createL10NHook } from './create-l10n-injector';
 import { getSupportedLocale, mapLocaleToIntlLocale } from './utils';
 
 type ImportData = {
   default: TimeZones;
 };
 
+type TranslationData = {
+  default: TimeZoneTranslations;
+};
+/**
+ * TODO:
+ * - handle fallback to 'default'/core.json when there are no translations for a locale
```

Comment:
There are different ways to handle this.

For example, we could remove all "empty" locale files and omit the locale in the switch-case, so that the fallback/default `en` locale is used.

This also means that at least the `en.json` should be defined

Reply:
I was thinking that we could lazy-load the locale first, and if the length `locale.default` is zero then lazy-load `en`, that way we wouldn't _necessarily_ always have to define `en.json`